searvey/coops.py

```diff
@@ -70,9 +70,9 @@ def product(
         self,
         product: StationDataProduct,
         start_date: datetime,
-        end_date: datetime = None,
```

Comment:
@pmav99 this type of typing seems to be available for Python `3.10` and up (https://peps.python.org/pep-0604/). Aren't we supporting `3.8` and `3.9`?

Reply:
This is correct. Nevertheless, if you add `from __future__ import annotations` at the beginning of the file then this syntax is supported. Try this:

``` python
from __future__ import annotations

a: int | None = None
```
save it to a file and execute it with python3.8. Both with the future import and without. With the import there is no problem. Without you get a `TypeError`.

Reply:
Oh, I missed the import! Thanks